stringer.go

```diff
@@ -53,7 +53,7 @@ var (
 	altValuesFunc   = flag.Bool("values", false, "if true, alternative string values method will be generated. Default: false")
 	output          = flag.String("output", "", "output file name; default srcdir/<type>_string.go")
 	transformMethod = flag.String("transform", "noop", "enum item name transformation method. Default: noop")
-	trimPrefix      = flag.String("trimprefix", "", "transform each item name by removing a prefix. Default: \"\"")
+	trimPrefix      = flag.String("trimprefix", "", "transform each item name by removing a prefix or comma separated list. Default: \"\"")
```

Comment:
the command needs to be re-run with --help to update the readout in the readme as well